process/block/shardblock.go

```diff
@@ -654,6 +662,13 @@ func (sp *shardProcessor) CommitBlock(
 		return err
 	}
 
+	headerMeta, err := sp.getLastNotarizedHdr(sharding.MetachainShardId)
+	if err != nil {
+		return err
+	}
+
+	sp.appStatusHandler.SetStringValue(core.MetricCrossCheckBlockHeight, fmt.Sprintf("meta %v", headerMeta.GetNonce()))
```

Comment:
might use %d when you know you going to display an int (int64, uint64, int32, uint32, byte and so on)

Reply:
Done. Instend of "%v" now i use "d" if i m goind to display int variables.

Reply:
not done

Reply:
Done